recipes/platform.interfaces/all/conanfile.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+
+class PlatformInterfacesConan(ConanFile):
+    name = "platform.interfaces"
+    license = "The Unlicense"
+    homepage = "https://github.com/linksplatform/Interfaces"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = """platform.interfaces is one of the libraries of the LinksPlatform modular framework, which uses 
+    innovations from the C++20 standard, for easier use of static polymorphism. It also includes some auxiliary 
+    structures for more convenient work with containers."""
+    topics = ("platform", "concepts", "header-only")
+    settings = "os", "compiler", "build_type", "arch"
+
+    @property
+    def _source_subfolder(self):
+        return self.name

```

Comment:
```suggestion
        return "source_subfolder"
```

This is kind of a convention in all the recipes